azure/src/main/java/org/apache/iceberg/azure/AzureProperties.java

```diff
@@ -189,4 +202,12 @@ public Mono<AccessToken> getToken(TokenRequestContext request) {
       builder.endpoint("https://" + account);
     }
   }
+
+  public KeyWrapAlgorithm keyWrapAlgorithm() {
+    return KeyWrapAlgorithm.fromString(this.keyWrapAlgorithm);
+  }
+
+  public String keyVaultUri() {
+    return this.keyVaultUri;
+  }
```

Comment:
wdyt about making this return Optional<String>, similar to what we do for the `adlsReadBlockSize` and `adlsWriteBlockSize` functions above?